src/main/python/pybuilder_docker_too/__init__.py

```diff
@@ -123,12 +123,17 @@ def docker_push(project, logger, reactor: Reactor):
 
 @task
 def docker_run(project, logger, reactor: Reactor):
-    do_docker_run(project, logger, reactor)
+    do_docker_run_task(project, logger, reactor)
 
 
 @before("verify_tavern")
 @depends('prepare')
-def do_docker_run(project, logger, reactor: Reactor):
+def do_docker_run_task(project, logger, reactor: Reactor):
+    _run_docker_compose(project, logger)
```

Comment:
It would probably make more sense to do the conditional branching here it doesn't make a whole lot of sense to let people run both compose and a container right?